docs/tempo/website/configuration/_index.md

```diff
@@ -374,6 +374,11 @@ storage:
             # Example: "bucket_name: tempo"
             [bucket_name: <string>]
 
+            # Optional
+            # Example: "cache_control: "no-cache""
+            # See the GCS documentation for more detail: https://cloud.google.com/storage/docs/metadata
```

Comment:
Pointing the reader to the GCS documentation is good, but we should include a line here that briefly explains that the metadata is written to the object.

Same comment for below config setting.